### DISCUSSION TITLE
EEG Markers

### DISCUSSION THREAD

**Discussion by SR-gituser** [mjstarrett/Landmarks#27] [Q&A] [answered]
Hello, posting this topic to potentially understand how EEG markers can be utilized with landmarks.

An experimental task will be run using the landmarks application, and we would like to obtain some EEG data as well if possible.
Currently we plan to send manual triggers (at the start of each trial), however to increase precision in the data, automatic markers would be ideal. Are their any requirements to facilitate this with our EEG equipment/processing suite.

Appreciate any feedback on this topic :)

**Comment by mjstarrett** [comment, marked as answer]
A few years back (~2020) I had added functionality for BrainVision EEG systems. Have a look at `Assets/Landmarks/Scripts/BrainAmpManager.cs`. Any class that inherits from ExperimentTask.cs was able to add unique triggers (up to either 128 or 256 I think) automatically when configured and active. Here's an example from the NavigationTask.cs class shown via inspector. 
<img width="533" alt="Screenshot 2024-04-15 at 8 04 03 PM" src="https://github.com/mjstarrett/Landmarks/assets/15330251/caab00eb-a219-477b-a135-0adc8097de00">

Beyond the above information, I would need to dig into the code to provide more detail. Have a look and feel free to reach out again for any clarifications. The gist is that having a BrainAmpManager.cs in your scene should enable your ExperimentTasks to send triggers via a specified port.

**Reply by SR-gituser** [reply]
Brilliant, thank you for providing the info for this functionality :)
